src/xdocs/index.xml.vm

```diff
@@ -553,13 +535,6 @@
               <td><a href="http://qalab.sourceforge.net/">QALab Home Page</a></td>
               <td>Supports tracking Checkstyle statistics over time.</td>
             </tr>
-            <tr>
-              <td><a href="https://vim.sourceforge.io/">Vim editor</a></td>
-              <td>Xandy Johnson</td>
-              <td><a href="https://vim.sourceforge.io/scripts/script.php?script_id=448">
```

Comment:
https://www.vim.org/scripts/script.php?script_id=448

Reply:
First link at https://www.google.com/search?q=Vim+editor+checkstyle+plugin&oq=Vim+editor+checkstyle+plugin+&aqs=chrome..69i57j33i160l5j33i299.7770j0j7&client=ms-android-google&sourceid=chrome-mobile&ie=UTF-8

Reply:
I misunderstood above, I thought you checked and plugin was gone, so I did not look. Done.